cmd/agent/daemon/app/app.go

```diff
@@ -211,6 +211,10 @@ func (a *App) Run(ctx context.Context) error {
 		}
 		defer storageConn.Close()
 
+		if cfg.EBPFEventsEnabled {
+			exporters.ContainerEventsSender = state.NewClickhouseContainerEventsExporter(log, storageConn)
+		}
```

Comment:
Added support to send events to local clickhouse for troubleshooting purposes

Reply:
The issue here that now if both castai and clickhouse is enabled this will override events sending.
Few options:
1. Change ContainerEventsSender to slice and append this exporter.
2. Do not allow to use both castai and clickhouse at the same time.

Reply:
I went with the easy solution since I was using clickhouse locally only for testing purposes but I will extend it to support multiple exporters

Reply:
Added multiple exporters https://github.com/castai/kvisor/pull/494/commits/a5eea0f7d81c50509d255e1d2cab05d7ab4f6e6b